backend/src/common/types/cars/autoria_request_params.ts

```diff
@@ -0,0 +1,12 @@
+interface AutoriaRequestParams {
+  category_id: number;
+  price_ot?: string;
+  price_do?: string;
+  powerFrom?: string;
+  powerTo?: string;
+  power_name?: number;
+  engineVolumeFrom?: string;
+  engineVolumeTo?: string;
```

Comment:
Are you sure you need that many optional keys?
In your case for request, we can use `Partial<AutoriaRequestParams>`

Reply:
:arrow_double_up: :+1: